lib/security.c

```diff
@@ -61,7 +61,8 @@
 #include "strcase.h"
 #include "warnless.h"
 #include "strdup.h"
-/* The last #include file should be: */
+/* The last #include files should be in this order */
+#include "curl_printf.h"
```

Comment:
Please add `#include <curl_memory.h>` as well, to make it use the "triplet" used in most other files. This one is needed to make the alloc functions use the (possibly) custom functions.

Reply:
Doh, of course. Fixed and pushed a rebased branch into this PR.